### PR TITLE
Fix code smells

### PR DESCRIPTION
**Pull request by jphillips2121**
Fix duplicated code blocks and string declarations

Resolves: BI-11551

### Type of change

* [X] New feature

### Pull request checklist

* [X] I have added unit tests for new code that I have added
* [X] I have added/updated functional tests where appropriate
* [X] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/go.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/golang/go/wiki/CodeReviewComments)__

### CODE REVIEW

File: service/gov_pay.go [hunk 1]
```diff
@@ -296,9 +300,10 @@ func callGovPay(gp *GovPayService, paymentResource *models.PaymentResourceRest)
 }
 
 func addGovPayHeaders(request *http.Request, paymentResource *models.PaymentResourceRest, gp *GovPayService) error {
-	treasuryBearer := "Bearer " + gp.PaymentService.Config.GovPayBearerTokenTreasury
-	chBearer := "Bearer " + gp.PaymentService.Config.GovPayBearerTokenChAccount
-	legacyBearer := "Bearer " + gp.PaymentService.Config.GovPayBearerTokenLegacy
+	var BearerToken = "Bearer "
```
Comment (emead) [inline service/gov_pay.go:303]: Use `:=` within functions

File: service/gov_pay.go [hunk 2]
```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/plutov/paypal/v4"
 )
 
+var GovPayRequestError = "error generating request for GovPay: [%s]"
```
Comment (emead) [inline service/gov_pay.go:18]: vars should begin with lower case if not exported

File: fixtures/refund.go [hunk 3]
```diff
@@ -2,6 +2,10 @@ package fixtures
 
 import "github.com/companieshouse/payments.api.ch.gov.uk/models"
 
+var LateFilingPenalty = "LATE FILING PENALTY"
+var RefundPending = "refund-pending"
+var PaymentSessionKind = "payment-session#payment-session"
```
Comment (emead) [inline fixtures/refund.go:7]: var names should begin with lower case letters if not exported

File: fixtures/refund.go [hunk 4]
```diff
@@ -2,6 +2,10 @@ package fixtures
 
 import "github.com/companieshouse/payments.api.ch.gov.uk/models"
 
+var LateFilingPenalty = "LATE FILING PENALTY"
```
Comment (emead) [inline fixtures/refund.go:5]: Original text was mixed case "Late Filing Penalty".
Is this change correct?

Comment (jphillips2121) [reply fixtures/refund.go]: It's used as test data so not breaking change but I'll change it back anyway as it makes more sense

**Review by emead — APPROVED** [review]

**Pull request merged**